### DISCUSSION TITLE
Anyone on the March beta and having issues?

### DISCUSSION THREAD

**Discussion by pdawg17** [cytech/Home-Assistant-wundergroundpws#91] [Q&A]
Since joining the March beta, all of my sensors are "unavailable". I thought my issue was an expired API key but I have regenerated it and still have same issue. Just trying to troubleshoot. I know issues can be expected on betas and am happy to report it to the beta team - I'm just not sure what the issue is and what to report.

**Comment by dataworx22** [comment]
All my sensors also unavailable, after I updated HA core to 2022.3.0. So I reverted my HA to a full backup of 18 February but sensors are still down, so I have no idea what happened.

**Reply by dataworx22** [reply]
Edit to above: I also regenerated my key prior to reverting to 18 Feb backup. Once I applied my new key my sensors all came back.... I'll stay away from updates for a bit.

**Comment by dataworx22** [comment]
I see on the developer's Github at: 
https://github.com/cytech/Home-Assistant-wundergroundpws

he states:
_UPGRADE NOTE: numeric_precision must be added to your exisiting configuration.yaml for this sensor. This adds the ability to display PWS data as integer or decimal. Valid options are 'none' or 'decimal'. See example below_.

I have added this to my config but it is invalid when checking the config and I am prevented from restarting the HA server.

Does this mean that my existing custom_components file has changed and I should replace it?

**Comment by cytech** [comment]
you need to make sure you have installed the latest wupws custom component release
0.8.1

**Reply by dataworx22** [reply]
I've updated accordingly to latest custom component, working like a champ again. Thanks very much for your response here.

**Reply by jmtatsch** [reply]
Yes thank you very much for the update as well.
Having this as a custom component on HACS as per my PR could make it easier to update this code for end users at almost no effort for you ;)

**Comment by cytech** [comment]
closed as outdated

**Comment by cytech** [comment]
closed as outdated
